needy/needy.py

```diff
@@ -21,6 +22,11 @@ def __init__(self, path, parameters):
 
         self.needs_directory = os.path.join(os.path.dirname(self.path), 'needs')
 
+    def jobs(self):
```

Comment:
again i'd suggest less vague terminology. even if we valued the historical significance of the "jobs" terminology enough to use it, i would want the extent of the impact to be in the argument / json parsing. historical significance doesn't need to root itself in our internals.

but in general, i don't value historical significance of things like this. viva la revolucion